### ISSUE TITLE
Wanted: List of EICAS messages

### ISSUE THREAD

**Issue by remenkemi** [Stratosphere-Studios/777-300ER#142]
Would like a list of all EICAS messages so we can check off the ones that have been completed.

**Comment by BRUHegg** [comment]
Just to clarify: you need all of the advisories, warnings and memo items?

**Comment by remenkemi** [comment]
Yes I think that would make it easier to plan and keep track of what we've done.

I have the same thing for the FMC notifications

**Comment by remenkemi** [comment]
https://quizlet.com/ph/698470127/b777-eicas-messages-flash-cards/
There's this quizlet lol

**Comment by BRUHegg** [comment]
Advisories:

- HYD PRESS(any possible system combination)
- FLIGHT CONTROLS
- HYD OVERHEAT PRI/DEM(any possible pump)
- HYD PRESS PRI/DEM(any possible pump)
- WINDOW FLT DECK(L/R)/WINDOWS
- SPOILERS
- STABILIZER(R/C)
- DISPLAY SELECT PNL
- EFIS CONTROL PNL(L/R)
- AUTOTHROTTLE DISC
- ALTITUDE ALERT
- NO AUTOLAND
- FLIGHT CONTROL MODE
- PRI FLIGHT COMPUTERS
- SPEEDBRAKE EXTENDED
- THRUST ASYM COMP
- GEAR DOOR
- BRAKE SOURCE
- STABILIZER CUTOUT
- AUTO SPEEDBRAKE
- TIRE PRESS
- BRAKE TEMP

Warnings

- STALL
- OVERSPEED
- AUTOPILOT DISC
- CONFIG SPOILERS
- CONFIG STABILIZER
- CONFIG PARKING BRAKE
- CONFIG GEAR STEERING
- CONFIG FLAPS
- CONFIG GEAR
- STABILIZER

Memo items:

- PARKING BRAKE SET
- SPEEDBRAKE ARMED
- DOORS AUTO

**Comment by remenkemi** [comment]
Thanks!

**Comment by remenkemi** [comment]
Wait is this just stuff you've implemented? Do you have a list of all messages?

**Comment by Capt120EK** [comment]
[] LANDING ALTITUDE - It is on when there is no FPL in the FMC

**Comment by SimulatedSkies** [comment]
Some more

1. ELEC GEN (Advisory)
2. FIRE ENG L/R (Warning)
3. RECIRC FANS OFF (Memo item)

Give me a few minutes I am screwing around in the PMDG 777 trying to get more warnings, I find it exponentially entertaining compared to looking at manuals.

**Comment by remenkemi** [comment]
Please don't flood this with replies, I just need a definitive file with a list of every message.
